acceptance/tests/ensure_version_file.rb

```diff
@@ -6,7 +6,7 @@
 
 test_name 'PA-466: Ensure version file is created on agent' do
 
-  skip_test 'requires version file which is created by AIO' if [:gem, :git].include?(@options[:type])
+  skip_test 'requires version file which is created by AIO' if @options[:type] != 'aio'
```

Comment:
for future reference this is one i'd like to move to the puppet-agent test suite once that gets off the ground (https://github.com/puppetlabs/puppet-agent/pull/909)